http/server.go

```diff
@@ -83,6 +83,11 @@ RESET:
 		next, err := stream.Recv()
 		if err != nil {
 			h.log.Warn("http_server", "random stream round failed", "err", err)
+			h.pendingLk.Lock()
+			h.latestRound = 0
+			h.pendingLk.Unlock()
+			// backoff on failures a bit to not fall into a tight loop.
+			time.Sleep(50 * time.Millisecond)
```

Comment:
Should it be an exp. backoff at some point ? if so let's make a todo / issue so we not forget

Reply:
dunno if i really should be exponential - calls to watch / get randomness will fail if the node you're talking to hasn't made its beacon yet. the underlying grpc client will do a network backoff in trying to connect separate from this, so this is more about the drand node itself getting HTTP requests before its in beacon-producing mode. It would be annoying if the node was running for a bit before the ceremony occured, and as a result it took it any real time after randomness was available before it noticed

Reply:
Are you trying every 50ms to get the randomness from the drand node here, as long as the drand node didn't start the randomness generation ? If so, that seems to be okay to wait a bit more, if the randomness generation didnt started yet, nodes shouldn't get on the hook to get the randomness at a 50ms precision - drand's not meant to be that time granular.
Or am i missing stg ? Not entirely clear to me.